lib/url_parser.rb

```diff
@@ -168,7 +168,13 @@ def remove_extra_segments
   end
 
   def remove_git_extension
-    url.gsub!(/(\.git|\/)$/i, '')
+    operand = if url.is_a?(Array)
+                url
+              else
+                [url]
+              end
```

Comment:
```suggestion
    operand = Array(url)
```

I think this can be simplified to just using `Array()` to convert non array URLs and leave existing ones alone.